package.json

```diff
@@ -18,6 +18,7 @@
     "sf:e2e:headless:ci:affected": "npx nx print-affected --select=projects | grep -E -v 'storefront' && echo 'nothing changed, skip' || npm run sf:e2e:headless:ci",
     "release": "run-s release:*",
     "release-labs": "cd libs && npm run release-labs",
+    "release-patch": "cd libs && npm run release-patch",
     "release:libs": "cd libs && npm run release",
```

Comment:
`release:libs` -> `release-libs` with a dash to be consistent? 
or it was done on purpose?

Reply:
or maybe to change `release-labs` to `release:labs`?